api/datastore/backup.go

```diff
@@ -43,11 +43,12 @@ func (store *Store) Restore() error {
 }
 
 func (store *Store) RestoreFromFile(backupFilename string) error {
+	store.Close()
```

Comment:
sttore is being closed here but I dont see code to reopen the connection

Reply:
Straight after the Copy below